woocommerce/storefront-product-sharing
# 📦 NEW: Add build tools (#14)

Fixes #14

Add build tools to generate CSS and translations files.

### Screenshots

n/a

### How to test the changes in this Pull Request:

1. Check out this PR
2. Run `npm i` to install all dependencies → Dependencies should be installed
3. Run `npm run build` to build the extension → The file `storefront-product-sharing.zip` should be created in root
4. Run `npm run build:dev` to render JS and SCSS files → The files `assets/css/style.css`, `assets/css/style.css.map` & `assets/css/style.min.css` will be rendered
5. Run `npm run start` to watch SCSS changes → If `src/css/style.scss` will be changes, the files `assets/css/style.css`, `assets/css/style.css.map` & `assets/css/style.min.css` will be rendered
6. Run `npm run watch` to watch SCSS changes → If `src/css/style.scss` will be changes, the files `assets/css/style.css`, `assets/css/style.css.map` & `assets/css/style.min.css` will be rendered
7. Run `npm run js` to render JS files → As this repo does not contain JS files, this will not deliver a result
8. Run `npm run css` to render SCSS files → The files `assets/css/style.css`, `assets/css/style.css.map` & `assets/css/style.min.css` will be rendered
9. Run `npm run makepot` to render the POT file → The file `languages/storefront-product-sharing.pot` will be rendered

### Changelog

> Dev - Add build tools to generate CSS and translations files. #14

### Tests

n/a

### Note

Due to insufficient permissions, I wasn't able to send this PR against this repo. Therefore, I had to send the PR via my forked repo.

cc: @jconroy (Pinging you here as I'm also unable to request a review from you.)
